charts/opentelemetry-collector/values.yaml

```
@@ -361,6 +361,24 @@ presets:
       #  - selector: route() where attributes["service.name"] == "one"
       #    histogramBuckets: []
 
+  # Span Metrics Sanitization wires the redaction processor into the traces pipeline to scrub
+  # high-cardinality span names/HTTP URLs and sanitize database statements whenever span metrics
+  # presets are enabled.
+  # - By default `enabled` is `null`, so the chart turns sanitization on automatically whenever
+  #   `presets.spanMetrics.enabled` or `presets.spanMetricsMulti.enabled` is true.
+  # - Set `enabled: false` to opt out even when span metrics are enabled, or `enabled: true` to
+  #   force sanitization on.
+  # - `sanitize_url_from_span_name` (default true) controls whether URL-like span names and URL
+  #   attributes are normalized (for example `GET /api/users/42/profile` → `GET /api/users/?/profile`).
+  # - `excludeDatabases` skips query sanitization for specific backends (valid: sql, redis,
+  #   memcached, mongo, opensearch, es); sanitized statements replace literals with placeholders
+  #   such as `SELECT * FROM orders WHERE user_id = 42` → `SELECT * FROM orders WHERE user_id = ?`.
+  # Additional redaction settings can be supplied via `extraConfig.processors.redaction`.
+  spanMetricsSanitization:
+    enabled: null  # auto: enabled when spanMetrics or spanMetricsMulti presets are enabled
+    sanitize_url_from_span_name: true
```

sanitize_url_from_span_name
```suggestion
    sanitize_url: true
```

---

you forgot about this